src/webrtc/callFeed.ts

```diff
@@ -129,9 +167,19 @@ export class CallFeed extends TypedEventEmitter<CallFeedEvent, EventHandlerMap>
         this.analyser.fftSize = 512;
         this.analyser.smoothingTimeConstant = 0.1;
 
-        const mediaStreamAudioSourceNode = this.audioContext.createMediaStreamSource(this.stream);
+        this.secondStream = this.stream.clone();
+        const mediaStreamAudioSourceNode = this.audioContext.createMediaStreamSource(this.secondStream);
         mediaStreamAudioSourceNode.connect(this.analyser);
 
+        const streamNode = this.audioContext.createMediaStreamSource(this.stream);
+        this.audioDelay = this.audioContext.createDelay(0.001);
```

Comment:
Could the delay be a `const` please?

Reply:
After some testing, this seems to be another quirk that this delaynode has. If I create a const delay, then it doesn't work.
I need to create a delay inside the class that can be overwritten with a new stream instead of creating a new one. 
Otherwise the connection between the nodes get lost and the old behavior with voice cutting off at the start is back.